website/app/container.server.ts

```diff
@@ -0,0 +1,46 @@
+import { createContainer, InjectionMode, asFunction, asValue } from 'awilix';
```

Comment:
Could be `modules/dependencies-context` or `modules/dep-injection` or similar?

Reply:
moved it to `modules` ;-)